installers/rpm-code/codes/common.sh

```diff
@@ -66,7 +72,8 @@ common_python() {
 
     # Lots of dependencies so we install here to avoid rpm collisions.
     # Slows down builds of pykern, but doesn't affect development.
-    codes_download pykern
+    # no quotes, because RADIA_RUN_BRANCH_PYKERN may be empty (no qualifier)
+    codes_download pykern ${RADIA_RUN_BRANCH_PYKERN:-}
```

Comment:
I don't think RADIA_RUN_BRANCH_PYKERN is used anywhere. If so, delete. This was an early attempt, I think.

Reply:
9969cf1